src/toffy/normalize.py

```diff
@@ -718,12 +718,35 @@ def normalize_fov(img_data, norm_vals, norm_dir, fov, channels, extreme_vals):
     else:
         os.makedirs(output_fov_dir)
 
-    # check if any values are outside expected range
-    extreme_mask = np.logical_or(norm_vals < extreme_vals[0], norm_vals > extreme_vals[1])
-    if np.any(extreme_mask):
+    # cap maximum normalization increase at 10X
+    abnormal_increase_mask = np.where(norm_vals < 0.1)
+    if len(abnormal_increase_mask) > 0:
+        abnormal_increase_chans = np.array(channels)[abnormal_increase_mask]
+        warnings.warn(
+            "The following channel(s) will suffer a >10X increase after normalization "
+            "for fov {}. Normalization capped at 10X for: {}.".format(fov, abnormal_increase_chans)
+        )
+        norm_vals[abnormal_increase_mask] = 0.1
+
+    decrease_mask = np.where(norm_vals > 1)
+    if len(decrease_mask) > 0:
+        decrease_chans = np.array(channels)[decrease_mask]
+        warnings.warn(
+            "The following channel(s) will suffer a decrease after normalization "
+            "for fov {}. Skipping normalization for: {}.".format(fov, decrease_chans)
+        )
+        norm_vals[decrease_mask] = 1.0
+
+    # check if any additional values are outside expected range
+    extreme_mask = np.where(np.logical_or(norm_vals < extreme_vals[0], norm_vals > extreme_vals[1]))
```

Comment:
What are the `extreme_vals` currently set to?

Reply:
`(0.4, 1.1)`. Had been meaning to ask if we should adjust these.

Reply:
@ngreenwald do we want to change the `extreme_vals` or is `(0.4, 1.1)` good?

Reply:
Yes, let's keep it here. Then add an additional warning for anything less than 0.1 saying it was capped at 0.1, and that the counts may be outside the range where normalization works

Reply:
Ok sounds good. We can probably include this inside the `warning` statement which indicates the values got capped at 10X normalization (aka norm_val capped at 0.1).